### ISSUE TITLE
Example .exe download fails

### ISSUE THREAD

**Issue by chrisjj** [eclipse-theia/theia#8265]
> First releases of theia-example v1.2.0 are now appearing here:
> https://download.eclipse.org/theia/That offers https://download.eclipse.org/theia/snapshots/windows/theia-Installer-1.2.0.exe 

Here (Chrome browser) this fails with "Access Forbidden".

Note: I would contribute this report on the issue itself "Set up CI system to create desktop installer of vanilla Theia #5481", except someone has locked that issue to non-"contributors" (as if testers aren't contributors) with "Pestering the project for one's pet feature is not productive."

**Comment by marcdumais-work** [comment]
> non-"contributors"

non-[committers](https://www.eclipse.org/membership/become_a_member/committer.php).
